### PR TITLE
chore(proposals): rotate execution closure proposal packet

### PR DESCRIPTION
**Pull request by jamesryancooper**
Policy reference: `.octon/framework/agency/practices/pull-request-standards.md`
Reminder: Run `@codex review`.

## What

Archives the older unified-execution atomic cutover proposal packet, adds the newer closure-certification packet and prompt set, and refreshes the generated proposal and extension/capability publication state that now points at the newer packet.

## Why

The local exploratory proposal workspace had both a superseded architecture packet and the newer closure-certification packet in flight. This PR moves the old packet out of the active path and updates the generated/read-model surfaces so proposal discovery reflects the newer packet. No-Issue: local proposal/archive refresh for the unified-execution closure packet.

## How

Deletes `.octon/inputs/exploratory/proposals/architecture/octon-unified-execution-constitution-cutover/**`, adds `.octon/inputs/exploratory/proposals/architecture/unified-execution-constitution-closure-certification-cutover/**`, registers the related prompts under `.octon/framework/scaffolding/practices/prompts/**`, and updates the generated proposal registry plus extension/capability publication receipts.

## Profile Selection Receipt

- Semantic version source(s): `version.txt`
- Release state (`pre-1.0` or `stable`): `pre-1.0`
- `change_profile` (`atomic` or `transitional`): `atomic`
- Hard-gate facts (downtime, coordination, migration/backfill, rollback, blast radius, compliance): proposal/archive maintenance only; no live runtime or authority mutation in this diff
- Tie-break status: no tie-break required
- `transitional_exception_note` (required when `pre-1.0` + `transitional`): not applicable

## Implementation Plan

- Workstreams and scope:
  - archive the superseded unified-execution proposal packet
  - add the closure-certification proposal packet and prompt entrypoints
  - refresh generated proposal and publication receipts
- Public interfaces/types/contracts affected:
  - exploratory proposal package layout
  - generated proposal registry
  - generated extension/capability publication receipts
- Test scenarios:
  - staged diff formatting
  - YAML integrity for generated state files
- Assumptions/defaults:
  - proposal/archive surfaces remain non-canonical inputs
  - generated state is expected to reflect the refreshed packet inventory

## Impact Map (code, tests, docs, contracts)

- Code: none
- Tests: `git diff --check`
- Docs: proposal packet markdown and prompt README entries
- Contracts: generated proposal registry and extension/capability publication receipts

## Compliance Receipt

- [x] Selected exactly one execution profile before planning/implementation.
- [x] Applied release-maturity gate from semantic version.
- [x] Enforced pre-1.0 default (`atomic`) unless hard gates required `transitional`.
- [x] Included `transitional_exception_note` when required.
- [x] Included tie-break escalation when applicable.
- [x] Updated impacted contracts/docs/tests.
- [x] Honored charter change-control constraint (no direct principles charter edits without override evidence).

## Exceptions/Escalations

none

## Tradeoffs

This is mostly proposal/archive and generated-state maintenance, so local verification is light. Review should focus on whether the archived/new packet split and generated registries are the intended current source-of-discovery state.

## Testing

- `git diff --check`

## Rollout

n/a

## Convivial Purpose Check

- [x] Feature expands genuine user capability (not synthetic engagement).
- [x] Attention and interruption behavior are justified and user-controllable.
- [x] No manipulative patterns or dark-pattern mechanics are introduced.
- [x] Data collection/extraction risk is minimal and explicitly justified.

## Risk Rubric

- Risk class: [ ] Trivial [ ] Low [x] Medium [ ] High
- Rollback plan: revert commit `4bc7f04dd`
- Rollback handle: `git revert 4bc7f04dd`
- Flags changed (name, owner, expiry, rollout): none
- Autonomy eligibility: [ ] autonomy:auto-merge [x] autonomy:no-automerge

## Contracts and Threat Model

- OpenAPI/JSON-Schema changes: none
- Threat model update/link: none

## Observability and Performance

- Traces/logs/metrics for changed flows: generated proposal/extension publication receipts only
- Representative traces for high risk changes: n/a
- Performance or bundle impact: none

## License and Provenance

- New dependencies and licenses: none
- Generated code/templates provenance notes: generated proposal/extension/capability receipt files refreshed alongside the packet move

## Checklist

- [x] Requirements met; edge cases handled
- [x] Security reviewed (authz, input validation, secrets)
- [x] Tests added or updated
- [x] Observability updated (logs, metrics, traces) if needed
- [x] No speculative abstractions or unnecessary complexity
- [x] Conventions followed; no drift introduced
- [x] Non-obvious decisions documented (comments, ADR)
- [x] All review conversations resolved

## Screenshots / Notes

No UI changes.

### CODE REVIEW

File: .octon/generated/proposals/registry.yml [hunk 1]
```diff
@@ -466,6 +466,41 @@ archived:
       - ".octon/state/evidence/runs/"
       - ".octon/state/continuity/repo/"
       - ".octon/generated/cognition/"
+  - id: "octon-unified-execution-constitution-cutover"
+    kind: "architecture"
+    scope: "octon-internal"
+    path: ".octon/inputs/exploratory/proposals/.archive/architecture/octon-unified-execution-constitution-cutover"
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .octon/generated/proposals/registry.yml:472]: The archive target now exists on this branch after merging `main` (`.octon/inputs/exploratory/proposals/.archive/architecture/octon-unified-execution-constitution-cutover/proposal.yml` and the full packet are present), so the generated registry entry now resolves to a real archived package.

File: .octon/framework/engine/runtime/crates/kernel/src/authorization.rs [hunk 2]
```diff
@@ -4318,6 +4331,29 @@ pub fn authorize_execution(
             Vec::new(),
         );
     }
+    if approval_required
+        && loaded_approval_grants.iter().any(|(grant, _)| {
+            grant.quorum_policy_ref.as_deref() != Some(canonical_quorum_policy_ref())
+        })
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .octon/framework/engine/runtime/crates/kernel/src/authorization.rs:4337]: Backfilled the canonical `quorum_policy_ref` into the historical wave4 approval request and grant in `8d6d8b32f`, so the legacy approved run stays compatible with the newer quorum-policy enforcement instead of falling closed.
